ONSdigital/sdc-responses-dashboard
# Move theme files to CDN

### Motivation and Context
Theme files doesn't need to live in the repo, hence it is now called from a CDN.

### What has changed
Removed theme css/js.
Have also added [SRI](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to external scripts.

### How to test?
- Look and feel of the app should be unchanged.

Make sure to run `yarn compile` before testing.
